wordpress-mobile/gutenberg-mobile
# Allow configuration of selection/caret color on Aztec view.

Fixes #1773 

This PR adds a prop to RCTAztecView to allow configuration of the selection/caret color

To test:
 - Change the code on one the block that uses the RichText component, and add a prop like this
` selectionColor= { 'orange' }`
 - Start the demo app 
 - Add the block changed
 - Check that the caret color and selection are correct.

PR submission checklist:
